modules/mobile/app/services/mobile/v0/appointments/service.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    module Appointments
+      # Connect's to VAMF's VA appointment and Community Care appointment services.
+      #
+      # @example create a new instance and call the get_appointment's endpoint
+      #   service = Mobile::V0::Appointments::Service.new(user)
+      #   response = service.get_appointments(start_date, end_date)
+      #
+      class Service < VAOS::SessionService
+        # Given a date range returns a list of VA appointments, including video appointments, and
+        # a list of Community Care appointments. The calls are made in parallel using the Typhoeus
+        # HTTP adapter.
+        #
+        # @start_date DateTime the start of the date range
+        # @end_date DateTime the end of the date range
+        # @use_cache Boolean whether or not to use the appointments cache within VAMF
+        #
+        # @return Hash two lists of appointments, va and cc (community care)
+        #
+        def get_appointments(start_date, end_date, use_cache = true)
+          params = {
+            startDate: start_date.utc.iso8601,
+            endDate: end_date.utc.iso8601,
+            pageSize: 0,
+            useCache: use_cache
+          }
+
+          responses = { cc: nil, va: nil }
+          errors = { cc: nil, va: nil }
+
+          config.parallel_connection.in_parallel do
+            responses[:cc], errors[:cc] = get(cc_url, params)
+            responses[:va], errors[:va] = get(va_url, params)
+          end
+
+          if errors.values.any? { |e| !e.nil? }
+            Rails.logger.error('mobile get va appointments call failed') if errors[:va]
+            Rails.logger.error('mobile get community care appointments call failed') if errors[:cc]
+            raise Common::Exceptions::BackendServiceException, 'VAOS_502'
```

Comment:
During my recent work implementing the VEText service, I discovered that this exception will bury the `VAOS_502` unless it is defined as a key in `config/locales/exceptions.en.yml` file.

Reply:
There's an entry for that key [here](https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/locales/exceptions.en.yml#L561-L566). I thought about adding our own key but it is marking that something happened to the upstream service and didn't want two different errors to map to the same issue.

Reply:
Cool, thanks for double-checking.